From 43fd142ed187fa1eccdaa01784d4dcf38e93d3c6 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Mon, 8 Oct 2018 11:25:28 +0800
Subject: [PATCH 4776/5242] MLK-19820 gpu: imx: dpu: Drop unnecessary const
 modifiers in struct dpu_devtype

commit  c311d71a893ced1394b2c4b5472962310ecccc3b from
https://source.codeaurora.org/external/imx/linux-imx.git

The driver creates constant objects with the type of struct dpu_devtype,
so it's unnecessary to use const modifiers for those non-pointer entries.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/imx/dpu/dpu-prv.h |   10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/gpu/imx/dpu/dpu-prv.h b/drivers/gpu/imx/dpu/dpu-prv.h
index ce7f899..ed3147c 100644
--- a/drivers/gpu/imx/dpu/dpu-prv.h
+++ b/drivers/gpu/imx/dpu/dpu-prv.h
@@ -181,7 +181,7 @@ struct cm_reg_ofs {
 };
 
 struct dpu_devtype {
-	const unsigned long cm_ofs;	/* common */
+	unsigned long cm_ofs;			/* common */
 	const struct dpu_unit *cfs;
 	const struct dpu_unit *decs;
 	const struct dpu_unit *eds;
@@ -197,19 +197,19 @@ struct dpu_devtype {
 	const struct dpu_unit *vss;
 	const struct cm_reg_ofs *cm_reg_ofs;
 	const unsigned int *intsteer_map;
-	const unsigned int intsteer_map_size;
+	unsigned int intsteer_map_size;
 	const unsigned long *unused_irq;
 	const unsigned int *sw2hw_irq_map;	/* NULL means linear */
 	const unsigned int *sw2hw_block_id_map;	/* NULL means linear */
 
-	const unsigned int syncmode_min_prate;	/* need pixel combiner, KHz */
-	const unsigned int singlemode_max_width;
+	unsigned int syncmode_min_prate;	/* need pixel combiner, KHz */
+	unsigned int singlemode_max_width;
 
 	/*
 	 * index:     0         1         2       3   4   5   6
 	 * source: fl0(sub0) fl1(sub0) fw2(sub0) fd0 fd1 fd2 fd3
 	 */
-	const u32 plane_src_na_mask;
+	u32 plane_src_na_mask;
 	bool has_capture;
 	bool has_prefetch;
 	bool has_disp_sel_clk;
-- 
1.7.9.5

